### DISCUSSION TITLE
problem with brew install mupen64plus

### DISCUSSION THREAD

**Discussion by ZivaOuPas** [Homebrew/discussions#3190] [Casks]
I can't install mupen64plus. Moreover, although the site https://github.com/mupen64plus/mupen64plus-core/releases/tag/2.5.9
does not seem to be very active, it looks like the later version is 2.5.9 and not 2.5.1 as proposed by brew.

==> Downloading https://ghcr.io/v2/homebrew/core/mupen64plus/manifests/2.5_1-1
Already downloaded: ~/Library/Caches/Homebrew/downloads/a42a5eda657939042d98291224ec28d0d62cdd6b86636cea79c6b583a4653d14--mupen64plus-2.5_1-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/mupen64plus/manifests/2.5_1-1
######################################################################## 100.0%
Error: Couldn't find manifest matching bottle checksum.
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/software_spec.rb:413:in `github_packages_manifest_resource_tab'
/usr/local/Homebrew/Library/Homebrew/software_spec.rb:366:in `fetch_tab'
/usr/local/Homebrew/Library/Homebrew/formula.rb:2395:in `fetch_bottle_tab'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:1180:in `fetch'
/usr/local/Homebrew/Library/Homebrew/install.rb:305:in `block in install_formulae'
/usr/local/Homebrew/Library/Homebrew/install.rb:278:in `map'
/usr/local/Homebrew/Library/Homebrew/install.rb:278:in `install_formulae'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:218:in `install'
/usr/local/Homebrew/Library/Homebrew/brew.rb:110:in `<main>'

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew will help you resolve the version issue.
